packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -0,0 +1,71 @@
+/* stylelint-disable  */
+
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
+
+%terra-demographics-banner {
+  background: $demographics-background;
+  color: #fff;
+  display: -ms-flex;
+  display: flex;
+  margin-bottom: 10px;
```

Comment:
We want to use rem values for units most of the time. We currently don't have a mixin for rems so calculations will need to be manual. For conversion, `14px` is assumed to be computed font size on the html element.

Here's a quick conversion for 10px when root font size is computed 14px.
`10px` = `0.714rem`